e2e/tests/components/file-upload/with-error.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, test } from '../../../fixtures'
+
+test.describe('file upload, with error', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/components/file-upload/with-error')
+  })
+
+  test('displays the error message', async ({ page }) => {
+    await expect(page.getByLabel('Upload a file', { exact: false })).toHaveDescription(
```

Comment:
`{ exact: false }` is the default in Playwright:

```suggestion
    await expect(page.getByLabel('Upload a file')).toHaveDescription(
```

Reply:
ahh my bad that was left over debugging